cmake/create_test.cmake

```diff
@@ -14,7 +14,9 @@ function(ginkgo_create_test test_name)
         target_link_libraries(${TEST_TARGET_NAME} PRIVATE "${GINKGO_CIRCULAR_DEPS_FLAGS}")
     endif()
     target_link_libraries(${TEST_TARGET_NAME} PRIVATE ginkgo GTest::Main GTest::GTest ${ARGN})
-    add_test(NAME ${REL_BINARY_DIR}/${test_name} COMMAND ${TEST_TARGET_NAME})
+    add_test(NAME ${REL_BINARY_DIR}/${test_name}
+        COMMAND ${TEST_TARGET_NAME}
+        WORKING_DIRECTORY "$<TARGET_FILE_DIR:ginkgo>")
```

Comment:
Either that, or use `ctest -C Release|Debug` since that uses the correct `WORKING_DIRECTORY` specified in `add_test`. As far as I can tell, that issue is only relevant to Windows, though.